### PR TITLE
add rules from fboldewin

### PR DESCRIPTION
**Pull request by mr-tz**
this includes and supersedes #310

### CODE REVIEW

File: data-manipulation/encryption/elliptic-curve/encrypt-data-using-curve25519.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 rule:
   meta:
     name: encrypt data using Curve25519
-    namespace: data-manipulation/encryption/curve25519
+    namespace: data-manipulation/encryption/elliptic-curve
```
Comment (mr-tz) [inline data-manipulation/encryption/elliptic-curve/encrypt-data-using-curve25519.yml:4]: unrelated to fboldewin's additions
graduating this rule 🥳

Comment (williballenthin) [inline data-manipulation/encryption/elliptic-curve/encrypt-data-using-curve25519.yml:4]: good catch

File: linking/runtime-linking/resolve-function-by-fin8-fasthash.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  features:
+    - or:
+      - basic block:
+        - and:
+            # 64-bit constants
+          - mnemonic: mov
+          - number: 0x880355F21E6D1965
+          - number: 0x2127599BF4325C37
+          - mnemonic: add
+      - or:
```
Comment (williballenthin) [inline linking/runtime-linking/resolve-function-by-fin8-fasthash.yml:23]: i think this `or` is redundant. but not a big deal.

**Review by williballenthin — APPROVED** [review]

**Comment by Ana06** [comment]
Was the `dont merge` label because of the v1.6.1 release? If so, we can remove it. 😉

**Comment by mr-tz** [comment]
I'll address https://github.com/fireeye/capa/issues/516 first to fix the linting issue.

File: linking/runtime-linking/resolve-function-by-fin8-fasthash.yml (hunk 2 above)
Comment (mr-tz) [reply linking/runtime-linking/resolve-function-by-fin8-fasthash.yml]: removed this

**Pull request merged**